### PR TITLE
[#4] Remove java agent installation and upgrade to molecule 2

### PR DESCRIPTION
**Pull request by sorobon**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
-## [Unreleased](https://github.com/idealista/newrelic-role/tree/develop)
+## [Unreleased](https://github.com/idealista/newrelic-servers-agent-role/tree/develop)
+### Changed
+- *Role and repository name changed to newrelic-servers-agent-role* @sorobon
+- *Migration to molecule 2.x*
```
Comment (dortegau) [inline CHANGELOG.md]: Who made the migration? 😜

File: meta/main.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 galaxy_info:
   company: Idealista
   min_ansible_version: 2.2
```
Comment (dortegau) [inline meta/main.yml]: I think the minimum version is 2.4.x.x now

File: tasks/main.yml [hunk 3]
```diff
@@ -10,6 +10,3 @@
 - include: config.yml
```
Comment (jnogol) [inline tasks/main.yml]: I would use `import_tasks`, as in here: https://github.com/idealista/airflow-role/blob/develop/tasks/main.yml

File: README.md [hunk 4]
```diff
@@ -21,17 +21,17 @@ These instructions will get you a copy of the role for your ansible playbook. On
...
 
 ### Installing
 
 Create or add to your roles dependency file (e.g requirements.yml):
 
 ```
-- src: idealista.newrelic-role
+- src: idealista.newrelic-servers-agent-role
   version: 1.0.0
   name: newrelic
```
Comment (jnogol) [inline README.md]: The name should be `newrelic-servers-agent`, as in L50 ;)

File: CHANGELOG.md (hunk 1 above)
Comment (sorobon) [reply CHANGELOG.md]: done :)

File: meta/main.yml (hunk 2 above)
Comment (sorobon) [reply meta/main.yml]: done

File: tasks/main.yml (hunk 3 above)
Comment (sorobon) [reply tasks/main.yml]: done

File: README.md (hunk 4 above)
Comment (sorobon) [reply README.md]: done

**Review by dortegau — APPROVED** [review]

**Pull request merged**